jerry-debugger/jerry-client-ws.py

```diff
@@ -728,6 +757,27 @@ def parse_source(debugger, data):
     else:
         logging.debug("No pending breakpoints")
 
+def printSource(self, line_num):
+    last_bp = self.debugger.last_breakpoint_hit
+    lines = last_bp.function.source
+    if last_bp:
```

Comment:
Btw we could do an early return here:

    if not last_bp:
      return

This way we don't need to indent the lines below.